teps/0128-scheduled-runs.md

```diff
@@ -63,7 +64,100 @@ If the EventListener is exposed outside of the cluster, the user must also sign
+      - name: git-url
+        value: "https://github.com/tektoncd/community.git"
+      workspaces:
+      - name: git-source
+        emptyDir: {}
+    schedule: "0 0 * * *"
+```
+
+`ScheduledTemplate` reconciler which creates resources will be part of the Tekton Triggers.
+
+### CLI
+
+As part of future work for this proposal, we can add a CLI command to manually trigger execution of a ScheduledTrigger
+similar to the kubectl functionality `kubectl create job --from=cronjob/my-cronjob`, for example:
+
+```sh
+tkn crontrigger start <scheduled_template_cr_name>
+```
+
+This functionality is tracked in https://github.com/tektoncd/cli/issues/1833.
+
+## Design Evaluation
+
+### Reusability
+
+This feature is essentially syntactic sugar for a CronJob creating a Tekton resource.
+It uses the functionality of TriggerTemplate but does not depends upon an EventListener.
+The schedule is a runtime concern, similar to other fields specified in EventListeners.
+
+### Simplicity
+
+The user experience with this proposal is much simpler with this proposal
+(a single additional line of configuration) than without it.
+For an example of what the user experience looks like without this proposal, see
+the [nightly release CronJob](https://github.com/tektoncd/plumbing/blob/be9826a5e75722782799e8094c3441295b185fe9/tekton/cronjobs/bases/release/trigger-with-uuid.yaml)
+used in the plumbing repo.
+
+### Flexibility
+
+- No new dependencies needed for this proposal
+- We are not coupling projects together but we add a new controller to Triggers.
+
+### Conformance
+
+- This proposal doesn't require the user to understand how the API is implemented,
+or introduce new Kubernetes concepts into the API.
+
+### Performance
+
+Creating a CronJob to simply create a new Tekton resource may not be very performant;
+however, performance of creating scheduled runs is not as important as (for example) performance
+of creating a run used for CI. Using CronJobs is a reasonable way to start, especially if it helps us avoid
+reimplementing cron syntax.
+
+### Security
+
+This proposal adds a new controller to the triggers with new permissions to create, update, and delete CronJobs.
+
+### Drawbacks
+
+- We might want to avoid setting a precedent of building functionality into Triggers to trigger on many different types
+of events. This proposal couples an event (a time occurring) with a Tekton resource creation, when we may prefer to keep
+events and resource creation separate. However, cron jobs are an extremely common simple use case that likely makes sense
+to address specifically.
+
+- We might want to build this functionality into a higher level API like [Workflows](./0098-workflows.md) or a larger
+feature like [polling runs](./0083-polling-runs-in-tekton.md). This proposal doesn't prevent that,
+and ScheduledTriggers could be leveraged to implement similar functionality in a higher level system.
+
+## Alternatives
+
+### ScheduledTrigger CRD with Triggers in Spec
```

Comment:
would you mind adding a few notes here about why this proposal was not chosen in favor of the new proposal? I think there was some detail here previously but it may have gotten removed.

Reply:
Sure.

Reply:
I think con cover this.